WebCssArchitecture.md

```diff
@@ -0,0 +1,78 @@
+CSS Architecture
+================
+
+Architectural principles
+------------------------
+
+- The UI-design principles(language) and CSS naming share a common structure
+- Mobile first, so the device with smallest screen has less css parsing
+- [Use immutable css objects](https://csswizardry.com/2015/03/immutable-css/)
+- [Single responsibility principle](https://en.wikipedia.org/wiki/Single_responsibility_principle)
+- Components have there own namespace so they can be refactored
+- Use predictable CSS rules
+- CSS must be self documenting
+- Keep the [specificity](https://css-tricks.com/specifics-on-css-specificity/) at 0,0,1,0 or 0,0,0,1
+- Max dept of applicability: 1
```

Comment:
`dept` => `depth`
This would be a big change for us to change to no nested classes. So can we please leave this out of here?